### PR TITLE
Adds iframe options to SDK

### LINKED ISSUE #1: Include a parameter for running in an iFrame
Per [this KB article](https://help.fullstory.com/using/recording-iframes), add a parameter to the init function that sets `window['_fs_run_in_iframe']`

### PR DESCRIPTION
**Pull request by van-fs**
This PR fixes #1 and adds control over the snippet's iFrame recording options as seen in [Can FullStory capture content that is presented in iframes](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +45,27 @@ FullStory.init({ orgId: '<your org id here>' });
...
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+
+  constructor() {
+    FullStory.init({ orgId: '<your org id here>', recordCrossDomainIFrames: true });
```
Comment (patrick-fs) [inline README.md]: I love the addition of the Angular example! Wondering though if we should include `recordCrossDomainIFrames` option given that we kind of want people to think twice about doing this (per our security notice). This is more of an optics thing. Would it be a good idea to remove it from the example? It's fairly well documented.

**Comment by patrick-fs** [comment]
This is much nicer! Check out the comment I left. Also: please commit a changelog entry and a minor version bump. Make sure to bump the version in `package.json` and `package-lock.json`

File: README.md (hunk 1 above)
Comment (van-fs) [reply README.md]: Switched to `debug` to play it safe.

**Comment by van-fs** [comment]
OK, I think we're completed.  I bumped to a new minor release since we've added new options.

**Pull request merged**